subsys/net/ip/net_context.c

```diff
@@ -1016,7 +1016,11 @@ static int create_udp_packet(struct net_context *context,
 
 		pkt = tmp;
 
-		r = net_ipv6_finalize(pkt, net_context_get_ip_proto(context));
+		net_pkt_compact(pkt);
```

Comment:
Oh no, net_pkt_compact() is still there? ;-)

Reply:
yes still part of the legacy api (create_udp_packet() here is also legacy btw)